R/utils.R

```diff
@@ -211,23 +210,32 @@ beautify <- function(x, standardized = FALSE, use_math = FALSE, ...) {
       }, ...))
       variable_label(tmp) <- variable_label(y[[i]])
       y[[i]] <- tmp
-    } else if (i == "estimate"){
+    } else if (i == "estimate") {
       args$x <- y[[i]]
-      y[[i]] <- do.call("printnum", args)
+      y[[i]] <- do.call("print_num", args)
+    } else if (i == "statistic") {
+      args_stat$x <- y[[i]]
+      y[[i]] <- do.call("print_num", args_stat)
     } else if (i == "term"){
       y[[i]] <- beautify_terms(as.character(y[[i]]), standardized = standardized)
+    } else if (i == "model"){
+      tmp <- beautify_model(as.character(y[[i]]), standardized = standardized)
+      variable_label(tmp) <- variable_label(y[[i]])
+      y[[i]] <- tmp
     } else {
-      y[[i]] <- printnum(y[[i]])
+      y[[i]] <- print_num(y[[i]])
     }
   }
 
   # rearrange ----
   y <- sort_columns(y)
 
-  if(!is.null(y$term)) {
-    perm <- term_order(y$term)
+  identifier <- intersect(colnames(y), c("term", "model")) 
```

Comment:
`stopifnot(length(identifier) <= 1L)`? I know it should usually not happen that both `term` and `model` are present, but I think such a test will be helpful for developing new methods or refactoring old ones